optimade/server/config.py

```diff
@@ -1,6 +1,7 @@
 # pylint: disable=no-self-argument
```

Comment:
Should be removable since the other validator has been removed and the original root validator has this disable comment already in it's definition line.
```suggestion
```